internal/statemachine/image_definition.go

```diff
@@ -95,7 +95,7 @@ type UserDataType struct {
 
 // PPAType contains information about a public or private PPA
 type PPAType struct {
-	PPAName     string `yaml:"name"         json:"PPAName"`
+	PPAName     string `yaml:"name"         json:"PPAName"               jsonschema:"pattern=[a-zA-Z0-9]/[a-zA-Z0-9]"`
```

Comment:
I think you probably mean `pattern=^[a-zA-Z0-9]+/[a-zA-Z0-9]+$` ?

Reply:
ahahah, you're right! For a moment I was scratching my head why the validation still succeeds on such a pattern, but then I realized it's basically matching only the 'middle' only, so one character each side of the `/` sign.

Reply:
Even further than that, it seems -, +, . and _ are also valid according to launchpad.